ruby/puppet

```diff
@@ -1 +1 @@
-Subproject commit 37d1d457b38d29c39c689ddd484285c7821a7194
+Subproject commit 02328a9b5617c9e0746f9403bab099035ca96492
```

Comment:
So this commit is only in puppet#master, which doesn't seem good, since I think we are trying to keep puppet-server#stable in-sync with puppet#stable.  I might just set this to be the latest commit in puppet#stable.